src/System.ServiceModel.Primitives/tests/ServiceModel/DuplexChannelFactoryTest.cs

```diff
@@ -101,7 +101,8 @@ public static void CreateChannel_BasicHttpBinding_Throws_InvalidOperation()
             factory.CreateChannel();
         });
 
-        Assert.True(exception.Message.Contains("BasicHttpBinding"), "InvalidOperationException exception string should contain 'BasicHttpBinding'");
+        Assert.True(exception.Message.Contains("BasicHttpBinding"), 
+            string.Format("InvalidOperationException exception string should contain 'BasicHttpBinding'. Actual message:\r\n" + exception.ToString()));
     }
```

Comment:
At this point this is inconsistent since one place we check and the other we don't. I'm ok if we decide not to check the actual message string if we validate the exception type or atleast document the default and the anomaly.